examples/tree-view/src/reducers/index.js

```diff
@@ -1,65 +0,0 @@
-import { INCREMENT, ADD_CHILD, REMOVE_CHILD, CREATE_NODE, DELETE_NODE } from '../actions'
-
-const childIds = (state, action) => {
-  switch (action.type) {
-    case ADD_CHILD:
-      return [ ...state, action.childId ]
-    case REMOVE_CHILD:
-      return state.filter(id => id !== action.childId)
-    default:
-      return state
-  }
-}
-
-const node = (state, action) => {
-  switch (action.type) {
-    case CREATE_NODE:
-      return {
-        id: action.nodeId,
-        counter: 0,
-        childIds: []
-      }
-    case INCREMENT:
-      return {
-        ...state,
-        counter: state.counter + 1
-      }
-    case ADD_CHILD:
-    case REMOVE_CHILD:
-      return {
-        ...state,
-        childIds: childIds(state.childIds, action)
-      }
-    default:
-      return state
-  }
-}
-
-const getAllDescendantIds = (state, nodeId) => (
```

Comment:
:speech_balloon: Ahh, so those functions existed in the original version, got it

Reply:
So it's okay for me to leave it as a separate function?

Reply:
Eh, sure.